src/features/applications/components/ApplicationReviewActions/ApplicationStatusBadge.tsx

```diff
@@ -0,0 +1,21 @@
+import type { ApplicationStatus } from "~/server/api/routers/applications/types";
+import { Badge } from "~/components/ui/Badge";
+
+type Props = {
+  status?: ApplicationStatus;
+  size?: "xs" | "md" | "lg";
+};
+
+enum StatusBadgeVariant {
+  pending = "info",
+  approved = "success",
+  rejected = "warning",
+}
+
+export function ApplicationStatusBadge({ status, size = "lg" }: Props) {
+  return (
+    <Badge variant={status && StatusBadgeVariant[status]} size={size}>
+      {status}
+    </Badge>
+  );
+}
```

Comment:
This component is unnecessary, just render the Badge directly instead.

Reply:
done